### ISSUE TITLE
Self hosted dashboard? 

### ISSUE THREAD

**Issue by rallisf1** [inlang/inlang#62]
Hello,

Since the files are all here, it would be nice to have a setup documentation for them. Better yet if you packed the dashboard components under a single docker-compose project.

`inlang.config.json` also needs a server host field

**Comment by samuelstroschein** [comment]
Hey rallis, 

we want to offer self-hosting but not until our dashboard/APIs are stable. Offering a self-hosted version right now would lead to a lot of work from our site to constantly keep the documentation up to date and help with issues.  

I am moving this post to discussions, so other users can vote/discuss this "feature request".
